scripts/release_info.py

```diff
@@ -291,9 +285,17 @@ def main():
     setup_changes = _check_setup_scripts(current_release)
     storage_changes = _check_storage_models(current_release)
 
-    pr_numbers = _extract_pr_numbers(new_release_logs)
-    prs = get_prs_from_pr_numbers(pr_numbers, personal_access_token)
-    categorized_pr_titles = get_changelog_categories(prs)
+    if parsed_args.personal_access_token:
+        personal_access_token = parsed_args.personal_access_token
+        pr_numbers = _extract_pr_numbers(new_release_logs)
+        prs = get_prs_from_pr_numbers(pr_numbers, personal_access_token)
+        categorized_pr_titles = get_changelog_categories(prs)
+    else:
+        print('No personal access token provided, uncategorized changelog'
```

Comment:
Do you mean "no changelog" or "uncategorized changelog"? The printing of the changelog seems to be dependent on `if parsed_args.personal_access_token:`.

Reply:
I rephrased it, does it sound better now? 

I actually meant to say just the commit history will be output (which I referred to as the uncategorized changelog). Then realized that term is confusing.